contracts/FindMarket.cdc

```diff
@@ -974,13 +1086,18 @@ pub contract FindMarket {
 					}
 				}
 				if containsListingType && containsNFTType {
-					return AllowedListing(listingType: marketType, ftTypes: allowedFTTypes, status: item.status)
```

Comment:
getAllowed listing instead of returning at first hit, it goes through all the saleItems and append all possible FT for the listing type and then dedup it before returning. 
That means we can have 2 separate saleItems that allows same type of NFT 
e.g. 
saleItem 1 : listing : sale, FT : DUC, NFT : Dandy 
saleItem 2 : listing : sale. FT : FUT, NFT : Dandy 

These 2 FT will now be returned together when allowed listing is called. 
Before the change it will only return DUC because it is the first hit.